staging/src/k8s.io/dynamic-resource-allocation/kubeletplugin/draplugin.go

```diff
@@ -55,24 +54,21 @@ type DRAPlugin interface {
 	// after it returns before all information is actually written
 	// to the API server.
 	//
-	// The caller must not modify the content after the call.
+	// Invalid ResourceSlices will be rejected by the apiserver during
```

Comment:
```suggestion
	//
	// It is the responsibility of the caller to ensure that the pools and
	// slices described in the driver resources parameters are valid
	// according to the restrictions defined in the resource.k8s.io API.
	//
	// Invalid ResourceSlices will be rejected by the apiserver during
```

Other than that it looks good to me. I'd like to leave the formal LGTM to @klueska to get a second opinion on this text because I wrote it and shouldn't self-review.

Reply:
/lgtm

Reply:
done